tests/build.gradle

```diff
@@ -315,37 +330,11 @@ def loadWhiskProps(){
     p
 }
 
-/**
- * Aggregates the scoverage xml reports from various modules into a
- * single report
- */
-task aggregateCoverage(type: JavaExec, dependsOn: reportCoverage) {
```

Comment:
This task was never used as we rely on [codecov][1] to perform actual aggregation

[1]: https://codecov.io/